### PR TITLE
NPSP: update migration timeout message

### LINKED ISSUE #4806: Advanced Mapping: Migration page timeout message
**Issue:**
When turning on Advanced Mapping and the migration deployment hits the page timeout limit, we display a warning message informing the user that the page has timed out and they can try again immediately. The toggle for Advanced Mapping is disabled because the deployment is actually still in the queue. The warning message is slightly inaccurate and not advisable.

timeout

**Steps to recreate:**

1. Go to NPSP Settings > System Tools > Data Import Advanced Mapping
2. Trigger a long deployment of some kind
3. Click 'Advanced Mapping' toggle

### PR DESCRIPTION
**Pull request by psadie-sfdo**
- Updates the message thrown when help migration times out to include a link to the deployment status.

# Critical Changes

# Changes

# Issues Closed
- Fixes #4806 
# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by psadie-sfdo** [comment]
**lurch: attach W-036516

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001CBWfQAO" target="_blank">W-036516</a>

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -3130,6 +3130,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
...
         <value>Modifying this Object Group is taking longer than expected.</value>
     </labels>
+    <labels>
+        <fullName>bdiMigrationLongDeployment</fullName>
+        <categories>Advanced Mapping</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Error message for long deployment</shortDescription>
+        <value>Converting Help Text mappings to Advanced Mapping is taking longer than expected.</value>
+    </labels>
```
Comment (psadie-sfdo) [inline src/labels/CustomLabels.labels]: @melissabarber - just tagging you in the newly added label.

Comment (psadie-sfdo) [reply src/labels/CustomLabels.labels]: Screen Shot 2019-11-12 at 12 39 40 PM

**Pull request merged**